modules/stanford_icon_grid/js/stanford_icon_grid.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * Fontawesome field widget.
+ */
+
+(function ($) {
+  Drupal.behaviors.stanfordFontawesome = {
+    attach: function (context, settings) {
+      $('.stanford-fontawesome-icon').once(function () {
```

Comment:
once shouldn't be needed here as the attach + context option should only run on areas that change via ajax or on initial load. You can scope your selector using context.

eg: `$('.stanford-fontawesome-icon', context).find('option').each(function ... `